tests/draft3/ref.json

```diff
@@ -141,6 +141,32 @@
         ]
     },
     {
+        "description": "ref, containing other keywords",
+        "schema": {
+            "definitions": {
+                "reffed": { "type": "string" }
+            },
+            "properties": {
+                "foo": {
+                    "$ref": "#/definitions/reffed",
+                    "type": "integer"
```

Comment:
Think it also might be clearer to pick an example that isn't just conflicting keywords. E.g., using type `array` behind a `$ref` but having a `minLength` will ignore the `minLength` sibling.